oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -551,7 +573,11 @@ private static boolean pingComputeEngineMetadata(
             request,
             MetricsUtils.getGoogleCredentialsMetricsHeader(
                 RequestType.METADATA_SERVER_PING, CredentialTypeForMetrics.DO_NOT_SEND));
+
+        LoggingUtils.logRequest(request, LOGGER_PROVIDER, "Pinging Metadata Server");
         HttpResponse response = request.execute();
+        LoggingUtils.logResponse(
+            response, LOGGER_PROVIDER, "Received response from Metadata Server");
```

Comment:
IIRC, `pingComputeEngineMetadata` is called as part of one of the ADC checks. Would it be possible to log this only for debug/ trace level?